integration-tests/jdbc/src/main/java/org/apache/camel/quarkus/component/jdbc/CamelResource.java

```diff
@@ -27,12 +28,7 @@
 import jakarta.annotation.PostConstruct;
 import jakarta.enterprise.context.ApplicationScoped;
 import jakarta.inject.Inject;
-import jakarta.ws.rs.Consumes;
-import jakarta.ws.rs.GET;
-import jakarta.ws.rs.POST;
-import jakarta.ws.rs.Path;
-import jakarta.ws.rs.PathParam;
-import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.*;
```

Comment:
I guess this one is related to your IDE configuration. 
Can you please revert importing all the package?